Libs/MRML/Core/vtkMRMLScalarVolumeDisplayNode.cxx

```diff
@@ -796,7 +796,7 @@ void vtkMRMLScalarVolumeDisplayNode::CalculateAutoLevels()
 
     window = max-min;
     level = 0.5*(max+min);
-    lower = this->GetLevel();
+    lower = range[0];
```

Comment:
Thanks for the explanation. The current behavior doesn't make much sense to me. Thresholding is disabled by default, so probably that's why it has not been causing problems.

@pieper @millerjv What do you think? Could have been a reason for using the previous level value as minimum threshold in vtkMRMLScalarVolumeDisplayNode::CalculateAutoLevels?